runner/app/live/trickle/media.py

```diff
@@ -26,8 +26,7 @@ async def run_subscribe(subscribe_url: str, image_callback, put_metadata, monito
         await asyncio.gather(subscribe_task, parse_task)
         logging.info("run_subscribe complete")
     except Exception as e:
-        logging.error(f"preprocess got error {e}", e)
-        raise e
+        logging.exception("run_subscribe got error", stack_info=True)
```

Comment:
why don't we want to print the actual error `e`?

Reply:
[logging.exception](https://docs.python.org/3/library/logging.html#logging.Logger.exception) will print it out, its magic